tests/features/autofailover.feature

```diff
@@ -122,6 +122,8 @@ Feature: Check pgconsul with disabled autofailover
 
         """
         Then <lock_type> "<lock_host>" has holder "pgconsul_postgresql1_1.pgconsul_pgconsul_net" for lock "/pgconsul/postgresql/leader"
+        And container "postgresql2" is in <replication_type> group
+        And container "postgresql3" is in <replication_type> group
         And <lock_type> "<lock_host>" has following values for key "/pgconsul/postgresql/replics_info"
```

Comment:
Do we still need this check now?
It is a repetition of `container "..." is in ... group` actually, but without check for `sync_state` field.